### PR TITLE
file for assignment 2

### PR DESCRIPTION
**Pull request by jmscarlett715**

### CODE REVIEW

**Comment by jmscarlett715** [comment]
@Brantron

File: assignment-2-ruby/jmscarlett/guessing_game.rb [hunk 1]
```diff
@@ -0,0 +1,110 @@
+def classic
+    random_number = Random.new.rand(20)
+loop do
+  print "Guess the number between 0 and 20 (e to exit): "
+  answer = gets.chomp
+  if answer == "e"
+    puts "The number was #{random_number}."
+    break
+  else
+    if answer.to_i == random_number
```
Comment (Brantron) [inline assignment-2-ruby/jmscarlett/guessing_game.rb:10]: doing something like `answer_to_int = answer.to_i` would help deduplicate these calls

File: assignment-2-ruby/jmscarlett/clean_house.rb [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+def wash_dishes
+  puts "The dishes have been washed."
+end
+
+def clean_toilet
+  puts "The toilet has been cleaned."
+end
+
+def clean_room(room)
+  if room == "bedroom"
```
Comment (Brantron) [inline assignment-2-ruby/jmscarlett/clean_house.rb:28]: a case statement here would be 🔥

File: assignment-2-ruby/jmscarlett/tiny_instacart.rb [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        sleep(SLEEP_DURATION)
+        puts "💵Payment received!💵"
+        puts "Please proceed to customer's location."
+        sleep(SLEEP_DURATION)
+        puts "Opening Google Maps to take you to customer's location..."
+        sleep(SLEEP_DURATION)
+        puts "*" * 40
+        sleep(SLEEP_DURATION)
+        sleep(SLEEP_DURATION)
+        sleep(SLEEP_DURATION)
```
Comment (Brantron) [inline assignment-2-ruby/jmscarlett/tiny_instacart.rb:109]: this feels like a lot of sleeps. could potentially do something like `sleep(LONG_SLEEP_DURATION)`

File: assignment-2-ruby/jmscarlett/tiny_instacart.rb [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+   sleep(SLEEP_DURATION)
+      sleep(SLEEP_DURATION)
+      puts "Have a great day, #{@shopper_name}!"
+      puts "🥕" * 20
+   sleep(SLEEP_DURATION)
+    end
+  
+    def get_answer
+      loop do
+        yield gets.chomp
```
Comment (Brantron) [inline assignment-2-ruby/jmscarlett/tiny_instacart.rb:143]: 🔥 🚒 🔥

**Review by Brantron — APPROVED** [review]

**Pull request merged**